src/Test.Utilities/AdditionalMetadataReferences.cs

```diff
@@ -36,6 +36,9 @@ public static class AdditionalMetadataReferences
         public static ReferenceAssemblies DefaultWithWinForms { get; } = Default
             .AddAssemblies(ImmutableArray.Create("System.Windows.Forms"));
 
+        public static ReferenceAssemblies DefaultWithoutCodeAnalysis { get; } = ReferenceAssemblies.Default
+            .AddAssemblies(ImmutableArray.Create("System.Xml.Data"));
```

Comment:
💭 At this point, would it make sense to just use `ReferenceAssemblies.Default` directly?

Reply:
Makes sense to me.